svc/contacts/zserver.go

```diff
@@ -0,0 +1,66 @@
+package contacts
+
+import (
+	"golang.org/x/net/context"
+
+	google_protobuf "github.com/golang/protobuf/ptypes/empty"
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/mysql"
```

Comment:
Why do not use `PostgreSQL` instead?

Reply:
Yes, this should be changed since we are using Postgres mostly. Originally I used mysql because I am most familiar with it.

Reply:
We will change this with the next release of `protoc-gen-gorm`, which should come around soon.